### DISCUSSION TITLE
Question regarding libraries update

### DISCUSSION THREAD

**Discussion by Nacurutu** [vladmandic/sdnext#15] [Q&A] [answered]
Hey @vladmandic 

May I ask, how can I achieve this on mine?

> Updated Python libraries to latest known compatible versions
> e.g. accelerate, transformers, numpy, etc.

What do I have to modify to always have the latest libraries versions?

Just updating the requirements_versions.txt will do the job?

Thanks in advance...

EDIT: Yes, it was just updating the requirements_versions.txt

Sorry to be so dumb regarding this kind of stuff :p

**Comment by vladmandic** [comment, marked as answer]
no worries.
and its not "latest", its "latest compatible" as some newer version break automatic - i've hand-picked those.
